docs/sources/reference/run.md

```diff
@@ -656,6 +656,11 @@ this driver.
 Default logging driver for Docker. Writes JSON messages to file. `docker logs`
 command is available only for this logging driver
 
+## Logging driver: syslog
+
+Syslog logging driver for Docker. Writes JSON messages to syslog. `docker logs`
```

Comment:
Was just about to say something similar. This document says it writes JSON, but the example output in the PR description isn't JSON.